src/badge/examples/EthereumYearBadge.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {Attestation} from "@eas/contracts/IEAS.sol";
+
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+
+import {ScrollBadge} from "../ScrollBadge.sol";
+import {ScrollBadgeAccessControl} from "../extensions/ScrollBadgeAccessControl.sol";
+import {ScrollBadgeCustomPayload} from "../extensions/ScrollBadgeCustomPayload.sol";
+import {ScrollBadgeNoExpiry} from "../extensions/ScrollBadgeNoExpiry.sol";
+import {ScrollBadgeNonRevocable} from "../extensions/ScrollBadgeNonRevocable.sol";
+import {ScrollBadgeSingleton} from "../extensions/ScrollBadgeSingleton.sol";
+
+string constant ETHEREUM_YEAR_BADGE_SCHEMA = "uint256 year";
```

Comment:
We could use a smaller int type to save an calldata cost.

Reply:
all `uint` types will pad to 32 bytes, so smaller int type won't save calldata cost.